_utilities/layout-grid.md

```diff
@@ -508,112 +414,249 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
-@include u-order($order);
-{% endhighlight %}
+<section id="utility-mixins" class="padding-top-4">
+  <h2 class="site-h2 margin-y-0">Utility mixins</h2>
+  {% include utilities/utility-mixin-intro.html %}
 
-### Example usage
-You can modify the variables to your custom values or just use the mixins with their default values. Here’s an example of using the default settings to create a two-column layout with a gap.
-
-{% highlight scss %}
-.example-container {
-  @include grid-container;
-}
-
-.example-row {
-  @include grid-row;
-
-  // Add column gaps
-  &.content-row {
-    @include grid-gap;
-  }
-}
-
-.example-content-main {
-  @include u-width(full);
-
-  @include at-media(tablet) {
-    @include grid-col(6);
-  }
-
-  @include at-media(desktop) {
-    @include grid-col(8);
-  }
-}
-
-.example-content-secondary {
-  @include u-width(full);
-
-  @include at-media(tablet) {
-    @include grid-col(6);
-  }
-
-  @include at-media(desktop) {
-    @include grid-col(4);
-  }
-}
-{% endhighlight %}
+<table class="usa-table--borderless site-table-responsive site-table-simple">
+    <thead>
+      <tr>
+        <th scope="col" class="tablet:maxw-card-lg display-inline-flex">Utility</th>
+        <th scope="col" class="display-inline-flex">Mixin</th>
+        <th scope="col" class="display-inline-flex">Example</th>
+      </tr>
+    </thead>
+    <tbody class="font-mono-2xs">
+      <tr>
+        <td data-title="Utility" class="tablet:maxw-card-lg">
+          <span>.grid-<code>container</code>
+          </span>
+        </td>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>container</code>
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-container
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:text-no-wrap tablet:maxw-card-lg display-inline-flex">
+          <span class="text-normal">
+            .grid-<code>row</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>row</code>
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-row
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:text-no-wrap tablet:maxw-card-lg display-inline-flex">
+          <span class="text-normal">
+            .grid-<code>col</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex">
+          <span>
+            grid-<code>col</code>(<a href="{{ site.baseurl }}/design-tokens/spacing-units/" class="token">units</a>)
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex">
+          <span>
+            @include grid-col(6)
```

Comment:
I think it would be helpful to show a few examples here
```suggestion
            @include grid-col
            @include grid-col(6)
            @include grid-col("auto")
```